src/movies.js

```diff
@@ -1,28 +1,70 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
+
+const movies = require("./data");
+
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+  const directors = moviesArray.map(function (element) {
+    return element.director;
+  })
+  return directors;
+}
```

Comment:
For an extra challenge try converting this to an arrow function, this way you can get shorter syntax instead of lines 8-10, you can have it in one ;)